README.rst

```diff
@@ -0,0 +1,14 @@
+===========
+nbgitpuller
+===========
+
+``nbgitpuller`` lets you distribute content in a git repository to your
+students by having them click a simple link. :ref:`Automatic, opinioned
```

Comment:
Perhaps it'd be best to focus on the student/receiver here since nbgitpuller doesn't require a teacher/instructor/leader and users can theoretically craft their own links:
> ``nbgitpuller`` lets users pull content from a git repository by clicking on a simple link.

Granted, the most common use case will probably be a teacher preparing both the material and the links for their students. Just food for thought, I've no strong opinion either way.